docs/proposal/aws-resource-handling.md

```diff
@@ -38,7 +38,11 @@ Where possible use [client tokens](https://docs.aws.amazon.com/AWSEC2/latest/API
 
 ## Tagging of resources
 
-Resources that are managed by the controllers/actuators should be tagged with: `kubernetes.io/cluster/<name or id>=owned` and `sigs.k8s.io/cluster-api-provider-aws=true`. The latter tag being used to differentiate from resources managed by other tools/components that make use of the common tag.
+Resources handled by these components fall into one of three categories:
+
+1. Fully-managed resources whose lifecycle is tied to the cluster. These resources should be tagged with `sigs.k8s.io/cluster-api-provider/<name or id>=owned` and `sigs.k8s.io/cluster-api-provider-aws/managed=true` and the actuator is expected to keep these resources as closely in sync with the spec as possible.
```

Comment:
Typo? `sigs.k8s.io/cluster-api-provider/`
should be `sigs.k8s.io/cluster-api-provider-aws/` throughout I think?

Reply:
I have not done a great job with consistency. The doc changes say one thing, and the code another 😄In code, I went with `sigs.k8s.io/cluster-api-provider-aws/cluster/<id>`, as a riff on the existing prefix: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/pkg/cloud/aws/tags/types.go#L87

Currently, that's used to define both `sigs.k8s.io/cluster-api-provider-aws/managed` (which this PR would supersede) and `sigs.k8s.io/cluster-api-provider-aws/role`, so adding a `/cluster/` felt fairly natural.

That said, I'm happy to adjust the tagging scheme to be `sigs.k8s.io/cluster-api/{role,cluster}` if that makes more sense to y'all.

Reply:
I think cluster-api-provider-aws is fine. It might make sense to standardize more as we refine the v1alpha2 and beyond milestones for cluster-api proper.